apps/01PigLatin.js

```diff
@@ -4,13 +4,26 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
 function pigLatin(word) {
-
-    // Your code here
-
+    var vowelIndex = 0;
+        while (!isVowel(word[vowelIndex])) {
```

Comment:
Anna, I want you to fix the indentation here. The rest of your code looks great!